### DISCUSSION TITLE
Why use home row mods for left and right split seperately?

### DISCUSSION THREAD

**Discussion by saitamandl** [urob/zmk-config#60] [Q&A] [answered]
When I was looking into the home row mods I got confused.
```
ZMK_HOLD_TAP(hrm,
    flavor = "balanced";
    tapping-term-ms = <HRM_TAPPING_TERM_MS>;
    quick-tap-ms = <HRM_QUICK_TAP_MS>;
    bindings = <&kp>, <&kp>;
)
```
Why not use unified hrm(preceding code)?
```
MAKE_HRM(hml, &kp, &kp, KEYS_R THUMBS)  // left-hand HRMs
MAKE_HRM(hmr, &kp, &kp, KEYS_L THUMBS)  // right-hand HRMs
```
Why using hrm for left and right hand split separately (preceding code)?

In ZMK [documentation](https://zmk.dev/docs/behaviors/hold-tap#hold-while-undecided-linger)  there is an explanation but I didn't understand that properly.

Would you please explain?

**Comment by urob** [comment, marked as answer]
Using `positional-hold-tap` makes it location specific. The readme at the root of this repo goes more into the motivation of why I think `positional-hold-tap`s are useful.
